src/test/java/uk/gov/di/ipv/cri/common/library/service/ConfigurationServiceTest.java

```diff
@@ -125,4 +146,63 @@ void shouldGetParameterByAbsoluteName() {
         assertEquals(PARAM_VALUE, configurationService.getParameterValueByAbsoluteName(PARAM_NAME));
         verify(mockSsmProvider).get(PARAM_NAME);
     }
+
+    @Test
+    @DisplayName("should cache ssm params and secrets manager secrets for 5 minutes by default")
+    void cacheFor5MinsByDefault() {
```

Comment:
we may as well keep the name of this unit test method in the same format as the others
e.g. shouldCacheFor5MinsByDefault